### PR TITLE
changes to the home page 

### PR DESCRIPTION
**Pull request by andresslc**

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -58,3 +58,8 @@ nav {
 .kpi__icon {
   max-height: 3rem; 
 }
+
+.test{
```
Comment (behilam) [inline assets/style.css]: Can you change it to a more descriptive class name?

File: pages/gender.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from distutils.command.config import config
```
Comment (behilam) [inline pages/gender.py]: I think this is not being used, or is it?

**Review by behilam — CHANGES_REQUESTED** [review]

File: assets/style.css (hunk 1 above)
Comment (andresslc) [reply assets/style.css]: change it to circle

File: pages/gender.py (hunk 2 above)
Comment (andresslc) [reply pages/gender.py]: done

**Review by behilam — APPROVED** [review]

**Pull request merged**